src/Microsoft.Diagnostics.Runtime.Utilities/DbgEng/DbgEngIDataReader.cs

```diff
@@ -187,6 +187,7 @@ ImageFileMachine.ARM or
             (ImageFileMachine)0x01c2 or                     // THUMB
             (ImageFileMachine)0x01c4 => Architecture.Arm,   // THUMB2
             (ImageFileMachine)0xAA64 => Architecture.Arm64, // ARM64
+            (ImageFileMachine)0x6264 => (Architecture)6 /* Architecture.LoongArch64 */, // LOONGARCH64
```

Comment:
This project is targeting net6.0, can we not use the name already?

Reply:
Ah, nvm. It was added in net7.0 and riscv64 was added recently in net9.0, so it can wait. All good. 👍

Reply:
That's right, `LoongArch64` was added in net7.0.